neofetch

```diff
@@ -1382,8 +1385,13 @@ get_model() {
 get_title() {
     user=${USER:-$(id -un || printf %s "${HOME/*\/}")}
 
-    case $title_fqdn in
-        on) hostname=$(hostname -f) ;;
+    case $os-$title_fqdn in
+        illumos-on|Solaris-on)
+            hostname=$(hostname)
+            domainname=$(domainname)
+            [[ -n "$domainname" ]] && hostname+=".$domainname"
+            ;;
+        *-on) hostname=$(hostname -f) ;;
         *)  hostname=${HOSTNAME:-$(hostname)} ;;
     esac
```

Comment:
Maybe we should change this to two case statements. I think it would make the logic clearer.

```bash
case $title_fqdn in
    on)
        case $os in
            illumos|Solaris)
                hostname=$(hostname)
                domainname=$(domainname)
                [[ -n "$domainname" ]] && hostname+=".$domainname"
            ;;
            *)
                hostname=$(hostname -f) 
            ;;
        esac
    ;;
    *)
        hostname=${HOSTNAME:-$(hostname)}
        hostname=${hostname%.*}
    ;;
esac
```